### ISSUE TITLE
New rule? Don't repeat enum values, default values or requiredness in the description.

### ISSUE THREAD

**Issue by veronicagg** [Azure/azure-openapi-validator#46]
Please find details at https://github.com/Azure/autorest/issues/1973

**Comment by veronicagg** [comment]
@sptramer is this something you'd like to take on your side?

**Comment by sptramer** [comment]
Yes, please create an issue on azure/azure-openapi-refchecker. This may require a bit of research because we would have to make sure that the keywords (default[s], value[s], etc.) are often related to already-stated information via NLP to reduce the number of false positives (which will encourage documentation writers to disable the rule.) As a result we may not implement it.
